### DISCUSSION TITLE
Deno Preact support?

### DISCUSSION THREAD

**Discussion by will-weiss** [puckeditor/puck#63] [General]
Amazing project! I'm very excited at the prospect of software creators and designers in particular being able to use a component library directly in this highly intuitive manner. Well done!

I'm curious what the path might look like in supporting preact, specifically for deno/fresh? 

Great concept, well executed — thanks for putting this out there!

**Comment by chrisvxd** [comment]
Hey @will-weiss, thanks for the kind words!

I've not used deno or fresh, but I can't see an _obvious_ reason as to why it wouldn't work as-is, besides some preact peer dependency warnings. Perhaps it's worth giving it a go and reporting back?

If you can get it working, it would be good to have it captured somewhere (at the very least documented in this issue, but maybe as an example application or added to the generator).

If there are any easy changes I can make to help, I'll see what I can do.

**Comment by chrisvxd** [comment]
I'm going to move this over to Discussions for the time being :)
